plugins/ruby-foreman-azure/debian/control

```diff
@@ -0,0 +1,13 @@
+Source: ruby-foreman-azure
+Section: ruby
+Priority: extra
+Maintainer: Daniel Lobato Garcia <elobatocs@gmail.com>
+Build-Depends: debhelper, foreman (>= 1.11.0), ruby-foreman-deface (<<2.0.0)
```

Comment:
missing foreman-assets, foreman-sqlite3 for build deps, in order to precompile assets (cause of the test failure)